helm/alfresco-content-services/values.yaml

```diff
@@ -466,6 +466,21 @@ filestore:
     timeoutSeconds: 10
     livenessPercent: 150
     livenessSavePeriodSeconds: 600
+  persistence:
+    # -- Persist filestore data
+    enabled: true
+    # -- Specify a storageClass for dynamic provisioning
+    storageClass:
+      enabled: true
+      accessModes:
+        - ReadWriteOnce
+      # -- Custom storage class name
+      name: null
+    # -- Use pre-provisioned volume (through claim) instead of dynamiv provisioning
+    existingClaim: null
+    data:
+      mountPath: "/tmp/Alfresco"
+      subPath: "alfresco-content-services/filestore-data"
```

Comment:
this is the first one but the object is to have different storage for each component, so may be worth adding all of those under a common filestore key?

Reply:
current chart already have `.persistence.component` keys. Here we move to a model where each component embeds its own persistence config in `component.persistence`